packages/avatars/src/components/Avatar.tsx

```diff
@@ -62,26 +62,15 @@ const Avatar = React.forwardRef(
   }
 );
 
-Avatar.propTypes = {
```

Comment:
@austin94, I recalled you had mentioned that `propType` checks are important in our components because not all of our consumers use TypeScript. I might have heard completely wrong, but could you elaborate on the heuristics of when `propType` checks should be removed (or included) when authoring a React component in TypeScript?

For example, in [this](https://github.com/zendeskgarden/react-containers/pull/108/files#diff-792be63542e4df272f763d73537a4f23R24) React component, the props are typed annotated, but also uses React's `propType` checking. @jzempel if you have an answer I'd love to hear your opinion too!

Reply:
I've restored this.